content/services/planning.md

```diff
@@ -171,13 +171,13 @@ summary: |
 
   Using an [object-oriented user experience strategy]
   (OOUX), we'll guide you through a
-  process to define the concepts or 
-  "objects" that make up your web app, 
-  the relationships between those objects, 
+  process to define the concepts or
+  "objects" that make up your web app,
+  the relationships between those objects,
   and the ways users want to interact with those objects.
   We will deliver an Object Map
-  to streamline communication 
-  between designers, developers, and stakeholders 
+  to streamline communication
+  between designers, developers, and stakeholders
   throughout the project.
 
   [object-oriented user experience strategy]: /authors/sondra/
```

Comment:
@SondraE It seems strange that this links to Sondra's page (I think I'd expect a link to something that would explain more about OOUX?), and Sondra's page says if you want to learn more you should check out this page.

Reply:
Ah yes. That R&C paragraph used to talk about having a specific OOUX strategist. Now that I've switch to talking about a general OOUX strategy, it does seem appropriate to remove the link from R&C to my author page.